packages/squiggle-lang/src/fr/plot.ts

```diff
@@ -183,6 +184,59 @@ export const library = [
 })`,
     ],
     definitions: [
+      makeDefinition(
+        [
+          frNamed(
+            "dists",
+            frOr(
```

Comment:
Maybe `frOr` should be below `frArray`? To allow mixed lists:

```
Plot.dists([
  2 to 3,
  { dist: 3 to 4, name: "second" }
])
```

Reply:
I tried this out. My intuition is that the typing would be confusing to users, and the need for this is quite rare.

That said, it would be very easy to add later on if anyone asks for it.